src/main/java/com/palantir/gradle/versions/VerifyLocksTask.java

```diff
@@ -74,45 +74,45 @@ public final void taskAction() throws IOException {
+    private void verifyLocksForScope(Function<LockState, SortedMap<MyModuleIdentifier, Line>> getterForScope) {
+        MapDifference<MyModuleIdentifier, Line> difference = Maps.difference(
+                getterForScope.apply(persistedLockState.get()), getterForScope.apply(currentLockState.get()));
 
         Set<MyModuleIdentifier> missing = difference.entriesOnlyOnLeft().keySet();
         if (!missing.isEmpty()) {
             throw new RuntimeException(
-                    "Locked dependencies missing from the resolution result: " + missing
+                    "Locked dependencies missing from the resolution result: "
+                            + missing
                             + ". Please run './gradlew --write-locks'.");
         }
 
         Set<MyModuleIdentifier> unknown = difference.entriesOnlyOnRight().keySet();
         if (!unknown.isEmpty()) {
             throw new RuntimeException(
-                    "Found dependencies that were not in the lock state: " + unknown
+                    "Found dependencies that were not in the lock state: "
+                            + unknown
                             + ". Please run './gradlew --write-locks'.");
         }
 
         Map<MyModuleIdentifier, ValueDifference<Line>> differing = difference.entriesDiffering();
         if (!differing.isEmpty()) {
-            throw new RuntimeException("Found dependencies whose dependents changed:\n"
-                    + formatDependencyDifferences(differing)
-                    + "\n\n"
-                    + "Please run './gradlew --write-locks'.");
+            throw new RuntimeException(
+                    "Found dependencies whose dependents changed:\n"
+                            + formatDependencyDifferences(differing)
+                            + "\n\n"
+                            + "Please run './gradlew --write-locks'.");
         }
     }
 
-    private static String formatDependencyDifferences(
-            Map<MyModuleIdentifier, ValueDifference<Line>> differing) {
-        return differing.entrySet()
-                .stream()
-                .map(diff -> String.format("" // to align strings
-                        + "-%s\n"
-                        + "+%s",
-                        diff.getValue().leftValue().stringRepresentation(),
-                        diff.getValue().rightValue().stringRepresentation()))
+    private static String formatDependencyDifferences(Map<MyModuleIdentifier, ValueDifference<Line>> differing) {
+        return differing.entrySet().stream()
+                .map(diff -> String.format(
+                        "" // to align strings
+                                + "-%s\n"
+                                + "+%s",
+                        diff.getValue().leftValue().stringRepresentation(), diff.getValue()
+                                .rightValue()
+                                .stringRepresentation()))
```

Comment:
This is not great, it took a bunch of format arguments and split only the last one.
This behaviour should only be triggered if all the previous arguments fit on one line.
Will track this.